google/cloud/monitoring_v3/services/metric_service/client.py

```diff
@@ -166,14 +166,14 @@ def transport(self) -> MetricServiceTransport:
         return self._transport
 
     '''@staticmethod
-    def metric_descriptor_path(project: str,) -> str:
+    def metric_descriptor_path(project: str,metric_descriptor: str,) -> str:
```

Comment:
This is a breaking change, unless this is one of the cases where there is another `metric_descriptor_path` below shadowng this one.  It seems likely that it is a *correct* change, but it does change behavior in a way which would cause a user to change her code.